modules/UI/recording/Recording.js

```diff
@@ -387,10 +387,15 @@ var Recording = {
      * @param show {true} to show the recording button, {false} to hide it
      */
     showRecordingButton (show) {
-        if (_isRecordingButtonEnabled() && show) {
-            $('#toolbar_button_record').css({display: "inline-block"});
+        let isVisible = show && _isRecordingButtonEnabled();
+        let id = 'toolbar_button_record';
+
+        console.log('recording is visible', isVisible);
+
+        if (isVisible) {
```

Comment:
The variable above should probably be renamed because  if (isVisible) showElement(); doesn't make sense. The console log above also prints the wrong message. The recording is _not_ visible at this point, it will be made visible after we call showElement.  Let's call the variable isShow.